### DISCUSSION TITLE
pls help lesson 5 on the image thing

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#50] [Q&A - iOS] [answered]
yes I have lots of errors cuz class fast and couldn't keep up plus slides very yes so ps help I add zip file yes thanks yes.
I got lost at the part where they were adding image
[trojan virus.zip](https://github.com/sst-inc/help/files/7409140/trojan.virus.zip)
s.

don't mind the name
also uh ye when I try to run it it just ends up as a blank white screen. I tried to switch around which view controller is the initial view controller but it never worked pls send help.

**Comment by Joeispro99** [comment, marked as answer]
Firstly, you have an error on line 96. To resolve this issue, go to line 12 and put extra delegates like UINavigationControllerDelegate and UIImagePickerControllerDelegate.
`class CollectionViewController: UICollectionViewController, UINavigationControllerDelegate, UIImagePickerControllerDelegate {`

Secondly, the reason why you only see a white screen is because in the Photos array, you don't have a single element inside which means no photos to display, so there will be 0 collection view cells displayed and hence there will be a white screen.

**Reply by sendhelpp** [reply]
when I try to do that I get a sigabrt error on this line:
let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reuseIdentifier, for: indexPath) as! PhotoCollectionViewCell

**Reply by Joeispro99** [reply]
@sendhelpp Delete line 21. It's from an old template and it causes the error.
I also redid the entire collection view controller as for some reason, it displayed the image but not as a collection view cell and now it works
[trojan virus.zip](https://github.com/sst-inc/help/files/7438774/trojan.virus.zip)
.

**Reply by granwyntan** [reply]
You could also comment it out, as shown in slide 25

**Reply by Ethan-Chew** [reply]
Hi, is this issue solved?

**Reply by sendhelpp** [reply]
ye sorry hadn't had time to look into it and reply as I had been busy with cca stuff for the past few days. should work, thanks :)

**Reply by sendhelpp** [reply]
uh wait but there is a problem with the moving around the image
[trojan virus.zip](https://github.com/sst-inc/help/files/7468991/trojan.virus.zip)
s

**Reply by sendhelpp** [reply]
nvm
